<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FourthRound</class>
 <widget class="QWidget" name="FourthRound">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2532</width>
    <height>1627</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>470</y>
     <width>760</width>
     <height>160</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>710</y>
     <width>321</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/Material/TextFrame.png);
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>1430</x>
     <y>710</y>
     <width>111</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/Material/correctButton.png);
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>720</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>240</y>
     <width>1201</width>
     <height>851</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/Material/backUi01.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>800</y>
     <width>600</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>1650</x>
     <y>840</y>
     <width>110</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>1610</x>
     <y>840</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(165, 122, 67);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>910</y>
     <width>1020</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
background-image: url(:/Material/countDown.png);
background-color: rgba(255, 255, 255, 0);
border-radius:12px;;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_3</zorder>
  <zorder>label_2</zorder>
  <zorder>label</zorder>
  <zorder>pushButton</zorder>
  <zorder>textEdit</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>label_6</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
